temp_library/trait.py

```diff
@@ -1,10 +1,9 @@
-"""skeleton for TextAdventure package"""
 import random
 
 
 class Trait(object):
-    def __init__(self, name, const, dice_num):
-        """rolled based trait
+    def __init__(self, name, const, dice_num=0):
+        """
             name: trait's name

```

Comment:
Please don't explain things with the same word you are trying to define. Also, don't explain things that are very easy to grasp from the code itself. I would prefer something more related to what is a Trait, why is there this class in the first place 😄